packages/table/src/react/onKeyDownTable.ts

```diff
@@ -22,6 +26,31 @@ export const onKeyDownTable: KeyboardHandler<TableConfig> = ({
 }) => {
   if (event.defaultPrevented) return;
 
+  const compositeKeyCode = 229;
+
+  if (
+    event.which === compositeKeyCode &&
```

Comment:
@12joan Is there a slate helper for that? `isComposing`?

Reply:
That's right. You can import `isComposing` from `@udecode/plate-common/react` and pass it the current `editor` to check if an IME composition is currently happening.

Reply:
I see. Can you explain this in a comment in the code, along with why an error occurs when typing with an IME while multiple rows are selected?

Reply:
A comment has been added, but I haven't found the specific reason yet 😂😂